azure-pipelines.yml

```diff
@@ -1,22 +1,22 @@
-pr: none
-
-trigger:
-- master
-
-jobs:
-
-- job: PublishToBintray
+-
```

Comment:
Hey @jtydhr88  can you add this to source-formatter.properties ignored files?  Because we don't want this one to be source formatted.